### DISCUSSION TITLE
Welcome to GameDevMind Discussions!

### DISCUSSION THREAD

**Discussion by gonglei007** [gonglei007/GameDevMind#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZQlby** [comment]
会建立相关群聊吗，这么有经验的前辈，对我一定很有帮助🌹

**Reply by gonglei007** [reply]
目前有一个QQ讨论群:242500383。也欢迎进群交流和分享。
